components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -1151,3 +1214,546 @@ func uniqueIps(ips []string) []string {
+		c.log.Debug("Started executing at %s \n", start.String())
+		c.writer.Println("Rotating PostgreSQL certificates")
+		pgRootCA := templateCerts.PostgreSQL.RootCA
+		c.writer.Printf("Fetching PostgreSQL RootCA from template %s \n", pgRootCA)
+		for _, pgIp := range templateCerts.PostgreSQL.IPS {
+			c.writer.Println("Rotating PostgreSQL follower node certificates")
+			err := c.rotatePGNodeCerts(infra, sshUtil, currentCertsInfo, pgRootCA, &pgIp, true)
+			if err != nil {
+				return err
+			}
+		}
+		timeElapsed := time.Since(start)
+		c.log.Debug("Time Elapsed to execute Postgresql certificate rotation since start %f \n", timeElapsed.Seconds())
+		// rotating OS certs
+		for i, osIp := range templateCerts.OpenSearch.IPS {
+			c.writer.Printf("Rotating OpenSearch node %d certificates \n", i)
+			err := c.rotateOSNodeCerts(infra, sshUtil, currentCertsInfo, &templateCerts.OpenSearch, &osIp, false)
+			if err != nil {
+				return err
+			}
+		}
+		timeElapsed = time.Since(start)
+		c.log.Debug("Time Elapsed to execute Opensearch certificate rotation since start %f \n", timeElapsed.Seconds())
+
+		// rotate AutomateCerts
+
+		for i, a2Ip := range templateCerts.Automate.IPS {
+			c.writer.Printf("Rotating Automate node %d certificates \n", i)
+			err := c.rotateAutomateNodeCerts(infra, sshUtil, currentCertsInfo, templateCerts, &a2Ip)
+			if err != nil {
+				return err
+			}
+		}
+
+		timeElapsed = time.Since(start)
+		c.log.Debug("Time Elapsed to execute Automate certificate rotation since start %f \n", timeElapsed.Seconds())
+
+		c.log.Debug("==========================================================")
+		c.log.Debug("Starting traffic on Autoamate nodes MAINTENANICE MODE OFF")
+		c.log.Debug("==========================================================")
+		err = startTrafficOnAutomateNode(infra, configRes, c.sshUtil, c.log)
+		if err != nil {
+			return err
+		}
+
+		for i, csIp := range templateCerts.ChefServer.IPS {
+			c.writer.Printf("Rotating Chef Server node %d certificates \n", i)
+			err := c.rotateChefServerNodeCerts(infra, sshUtil, currentCertsInfo, templateCerts, &csIp)
+			if err != nil {
+				return err
+			}
+		}
+
+		timeElapsed = time.Since(start)
+		c.log.Debug("Time Elapsed to execute ChefServer certificate rotation since start %f \n", timeElapsed.Seconds())
+
+		c.log.Debug("==========================================================")
+		c.log.Debug("Starting traffic on chef server nodes MAINTENANICE MODE OFF")
+		c.log.Debug("==========================================================")
+		err = startTrafficOnChefServerNode(infra, configRes, c.sshUtil, c.log)
+		if err != nil {
+			return err
+		}
+
+	}
+	return nil
+}
+
+func (c *certRotateFlow) rotatePGNodeCerts(infra *AutomateHAInfraDetails, sshUtil SSHUtil, currentCertsInfo *certShowCertificates, pgRootCA string, pgIps *IP, concurrent bool) error {
+	start := time.Now()
+	c.writer.Printf("Roating PostgreSQL node %s certificate at %s \n", pgIps.IP, start.String())
```

Comment:
In demo video there's a string in this print statement after date and time something like `m=+74....` what is that value is that expected?